modules/ROOT/pages/_partials/include-reauthentication-conditions.adoc

```diff
@@ -1,6 +1,6 @@
 Access Management prompts you to reauthenticate under the following conditions:
 
 * If more than 30 minutes have elapsed since you entered your password, Access Management prompts you to reenter your password.
-* If you have multi-factor authentication (MFA) enabled and more than 8 hours have elapsed since you entered your password and used MFA, Access Management prompts you to reenter your password and confirm the login using MFA.
+* If you have multi-factor authentication (MFA) enabled and more than 8 hours have elapsed since you entered your password and used MFA, Access Management prompts you to reenter your password and confirm the signin using MFA.
```

Comment:
```suggestion
* If you have multi-factor authentication (MFA) enabled and more than eight hours have elapsed since you entered your password and used MFA, Access Management prompts you to reenter your password and confirm the signin using MFA.
```